types/index.d.ts

```diff
@@ -328,6 +328,10 @@ interface DMLOptions {
   includeTriggerModifications?: boolean;
 }
 
+interface TransactionMethods<T> {
+  setIsolationLevel: (value: string) => T;
```

Comment:
Would be better to use an enum with all possible values, also runtime code should throw an error if unknown isolation level is passed.

Reply:
Well normally there are 4 levels, mssql has 5 (kinda 6), sqlite has 1, kinda 2, from my testing mysql (5.7) says it has 4 but really only works with 3. So I'm not sure exactly the best interface for it

Reply:
Supporting all 5 (or) 6 on validation level is fine, DB will throw an error if it doesn't support something.